server/api.go

```diff
@@ -347,10 +349,16 @@ func (a *API) connect(w http.ResponseWriter, r *http.Request) {
 		userID = a.p.GetBotUserID()
 	}
 
+	stateSuffix := ""
```

Comment:
The code prevented the "connect" button from the preference page to function, because it required a bot message. I changed the code to allow for both.